changelog/5025.feature.rst

```diff
@@ -0,0 +1,2 @@
+Added the ability to dynamically scale `sunpy.visualization.animator` instances.
+By specifying ``vmax="auto"``, it will now clip the minimum and maximum at each slider step to 1% and 90% respectively.
```

Comment:
We should make these limits symmetrical, e.g. use 1% and 99% or 10% and 90%. I think there's somewhere else in the library that we use 1% and 99%, so I would say lets go with that.

Reply:
@dstansby 99% causes the image to not be as visible but if that's the standard then I shall make that change.
I'll make the change for 2 specifying either of them separately.
While writing tests should I write 3 separate tests for ```vmax='auto'``` , ```vmin='auto'```, and both, or should I just have one test where both ```vmax='auto'``` and ```vmin='auto'```?